mixer/testdata/config/fluentd.yaml

```diff
@@ -0,0 +1,44 @@
+# Test with:
+# $ mkdir fluentd-log
+# $ docker run -d -p 24224:24224 -p 24224:24224/udp -v $PWD/fluentd-log:/fluentd/log fluent/fluentd
+
+# Configuration for logentry instances
+apiVersion: "config.istio.io/v1alpha2"
+kind: logentry
+metadata:
+  name: newlog
+  namespace: istio-system
+spec:
+  severity: '"warning"'
+  timestamp: request.time
+  variables:
+    source: source.labels["app"] | source.service | "unknown"
+    user: source.user | "unknown"
+    destination: destination.labels["app"] | destination.service | "unknown"
+    responseCode: response.code | 0
+    responseSize: response.size | 0
+    latency: response.duration | "0ms"
+  monitored_resource_type: '"UNSPECIFIED"'
+---
+# Configuration for a fluentd handler
+apiVersion: "config.istio.io/v1alpha2"
+kind: fluentd
+metadata:
+  name: fdhandler
+  namespace: istio-system
+spec:
+  address: "localhost:24224"
```

Comment:
this is the default address, right?

Reply:
Yes, I'd like to have is here in the test configs incase someone wants to copy it